### PR TITLE
Improved Credential Status Scheme

### PR DESCRIPTION
**Pull request by David-Chadwick**
This is an update of #145, which is superceded by this and therefore can be deleted

***
<a href="https://pr-preview.s3.amazonaws.com/David-Chadwick/vc-data-model/pull/176.html" title="Last updated on May 16, 2018, 7:07 PM GMT (f1c77e4)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/176/9ea3a8d...David-Chadwick:f1c77e4.html" title="Last updated on May 16, 2018, 7:07 PM GMT (f1c77e4)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -673,17 +673,22 @@ <h2>Expiration</h2>
...
         <h2>Status</h2>
-        <p>Credential status information, such as suspension or revocation,
-        may be provided by adding the following <a>property</a>:
+        <p>Information about the current status of a <a>verifiable credential</a>, 
+        such as suspension or revocation,
+        may be provided by adding the credentialStatus <a>property</a>. This property
+       comprises the type of credential status information that is being provided
+ (sometimes referred to as the credential status scheme), plus the id of the status
+type instance. The precise contents of the credential status information is
+           determined by the specific credentialStatus type definition, and will vary
```
Comment (msporny) [inline index.html]: nit: credentialStatus needs to be wrapped in &lt;code> elements.

File: index.html [hunk 2]
```diff
@@ -673,17 +673,22 @@ <h2>Expiration</h2>
 
       <section>
         <h2>Status</h2>
-        <p>Credential status information, such as suspension or revocation,
-        may be provided by adding the following <a>property</a>:
+        <p>Information about the current status of a <a>verifiable credential</a>, 
+        such as suspension or revocation,
+        may be provided by adding the credentialStatus <a>property</a>. This property
```
Comment (msporny) [inline index.html]: nit: credentialStatus needs to be wrapped in code tags: i.e. <code>credentialStatus<code>

Comment (David-Chadwick) [reply index.html]: done

File: index.html (hunk 1 above)
Comment (David-Chadwick) [reply index.html]: done

**Comment by msporny** [comment]
Good PR, no objections, merging.

**Pull request merged**